### ISSUE TITLE
Y24-327 - scRNA Core bulk test

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#1925]
**User story**
As a user of the scRNA Core pipeline, I would like to be confident that the pipeline will work for the volume of samples and size of pools that I'm planning to put through the end to end test.

**Who are the primary contacts for this story**
Abby, Andrew, Katy

**Who is the nominated tester for UAT**
This doesn't need UAT, it just needs the test to pass in the UAT and/or training envs.

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Discuss this story with Abby when she's back on 23/9/24. If she is planning to do a realistic test manually for UAT, then it's probably not worth creating this new automated test, and we should focus on completing the other stories.
- [ ] Create or update the scRNA Core Integration Suite tests to be close to what R&D are planning for their end to end test:
    - [ ] 10 samples per pool, 8 pools per chip
    - [ ] 24 samples per pool, 8 pools per chip

**Dependencies**
The development of the test is not blocked by the following, however we need to check it passes after the following are done:
- https://github.com/sanger/limber/issues/1909
- https://github.com/sanger/limber/issues/1914

**Comment by yoldas** [comment]
For recent UAT sessions, we have used the following integration suite branch: `scrna_core_pipeline_volume_test`, particularly the test `spec/limber/scrna_core_volume_spec.rb`

It allows specifying parameters for number of runs, number of vac tubes each run, etc. However it only covers the usual 'scRNA Core entry point 1 - LRC Blood Vac tubes' scenario. **It has all steps in separate methods** so disabling calls to them becomes easy in the main flow. 

I am looking into
- Well failing makes it difficult to track later stages and causes an error in full plates.
- Getting updates from the original scRNA Core tests, such as adding bulk submission step.

**Comment by yoldas** [comment]
I have written a volume test which covers Blood Banking, cDNA Prep and Library Prep. It can be configured using the let blocks. I have set the config to make 7 banking runs. From each run we pick 8 Seq and 2 Spare tubes. We also add 10 Input tubes to those to make 80 tubes going into pooling. 
GitLab MR: https://gitlab.internal.sanger.ac.uk/psd/integration-suite/-/merge_requests/177

Local Run: `ENVIRONMENT=local DEBUG_ON_FAIL=true bundle exec rspec spec/limber/scrna_core_volume_spec.rb`

Running it against uat and training environments needs Sequencescape UAT action for randomised FluidX barcodes to be deployed (merged into develop already).

**Comment by yoldas** [comment]
The Sequencescape UAT action for generating FluidX barcodes is deployed to UAT. I will run the bulk test against UAT using GitLab.
